curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc92ecdebaa1278505e87.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 656cc92ecdebaa1278505e87
+title: Task 106
+challengeType: 19
+dashedName: task-106
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Today, I'm going out. I can show you some places around here. Are you interested?
+-->
+
+# --description--
+
+`Places around` refers to locations in the nearby vicinity or area.
```

Comment:
I believe the expression should be "places around here", both in the description and text.